packages/flutter/lib/src/gestures/pointer_signal_resolver.dart

```diff
@@ -96,6 +96,9 @@ class PointerSignalResolver {
   void resolve(PointerSignalEvent event) {
     if (_firstRegisteredCallback == null) {
       assert(_currentEvent == null);
+      if (event is PointerScrollEvent) {
```

Comment:
It is strange to special-case scroll here, if we go down this path: Could all PointerSignalEvents be ignorable?

Reply:
In theory, yes. In practice the other two PointerSignalEvents are: `PointerScrollInertiaCancelEvent` (used to reset an overscroll effect, IIRC) and `PointerScaleEvent` (pinch to zoom); and I don't think the web emits either of those.

Reply:
(I didn't put it in the base interface for now because I didn't want to reimplement the same two methods 6 times, I thought 2 was enough for the PoC)

Reply:
This is not special-cased anymore.